addons/core/translations/en-us.yaml

```diff
@@ -60,6 +60,7 @@ questions:
   remove-confirm: Are you sure you want to remove this association?
   abandon-confirm: Are you sure you want to navigate away?  Your unsaved changes will be lost.
   retry-confirm: Would you like to retry?
+  clear-confirm: Are you sure you want to clear the resource id?
```

Comment:
I get a second modal that pops up for a fraction of a second when I hit clear:


https://github.com/hashicorp/boundary-ui/assets/5783847/b13d3cb7-6a1a-4d69-8af6-8934e8b26ee2

Reply:
thanks! updated this